src/index.html

```diff
@@ -5,8 +5,436 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-description">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">
+            Price:
+          </p>
+          <p class="card__price-value">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__description">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-description">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">
+            Price:
+          </p>
+          <p class="card__price-value">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__description">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-description">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">
+            Price:
+          </p>
+          <p class="card__price-value">
+            $2,199
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card">
```

Comment:
Will be better if you use tag article here instead of section
```suggestion
      <article class="card">
```